index.rst

```diff
@@ -1,40 +1,378 @@
-..
-  Content of technical report.
+:tocdepth: 1
 
-  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+Technical notes (hereafter, *technotes*) are a new documentation medium for `LSST Data Management <http://dm.lsst.org>`_ staff to publish stand-alone documents that are native to the web.
```

Comment:
Maybe say "(DM)" after "Data Management" here as "DM" is used later (yes, I know everyone who cares will know already).